tests/test_sensor.py

```diff
@@ -67,6 +69,49 @@ def status_func(value):
     assert sensor.status == Sensor.Status.NOMINAL
 
 
+@pytest.fixture
+def classic_observer():
+    def classic(sensor, reading):
+        pass
+
+    return create_autospec(classic)
+
+
+@pytest.fixture
+def change_aware_observer():
+    def change_aware(sensor, reading, old_reading):
+        pass
+
+    return create_autospec(change_aware)
+
+
+def test_observer_sorting(classic_observer, change_aware_observer):
+    """Check whether change-aware and classic ovserver callbacks are sorted appropriately."""
```

Comment:
```suggestion
    """Check whether change-aware and classic observer callbacks are sorted appropriately."""
```

Reply:
Not sure what you mean by "sorted". There is no guaranteed order of observer callbacks. Perhaps you mean "classified"?

Reply:
Yes, classified is probably a better word.